src/Sentry/Internal/Enricher.cs

```diff
@@ -82,6 +83,13 @@ public void Apply(IEventLike eventLike)
 
             // Default tags
             _options.ApplyDefaultTags(eventLike);
+
+            // Pass the runtime identifier as a tag also
+            const string ridTag = "runtime.identifier";
+            if (_runtimeLazy.Value.Identifier is { } ridValue && !eventLike.Tags.ContainsKey(ridTag))
+            {
+                eventLike.SetTag(ridTag, ridValue);
```

Comment:
No risk of this becoming a high cardinality value?

Runtime info isn't sent as a tag from the SDK, ingest extracts the tag at ingestion time. @jan-auer brought this up in the past, lets see if he has any pointers.

Reply:
Makes sense.  Let me know if there's a better place to put it.  Either way, it's very useful information that became available starting with .NET 5 that we haven't been including on events yet.